public/config/version.json

```diff
@@ -0,0 +1 @@
+{"sha":"72480175cdc7411835711287df7d0a48843232a0","ref":"feat-task-trials"}
```

Comment:
should this be here? I think it might have been written by running the code and we should add it to the .gitignore